common/src/main/java/com/wynntils/features/user/InfoMessageFilterFeature.java

```diff
@@ -321,6 +328,30 @@ public void onInfoMessage(ChatMessageReceivedEvent e) {
                 }
             }
 
+            if (horse != FilterType.KEEP) {
+                Matcher roomMatcher = NO_ROOM_PATTERN.matcher(msg);
+                if (roomMatcher.matches()) {
+                    e.setCanceled(true);
+                    if (horse == FilterType.HIDE) {
+                        return;
+                    }
+                    NotificationManager.queueMessage(
+                            new TextComponent("No room for a horse!").withStyle(ChatFormatting.DARK_RED));
```

Comment:
@kristofbolyai This is missing i18n. When checking the file, I notice that all of these notifications are missing i18n. We should probably do something about it.

I'll open a bug so we don't forget.

Reply:
No redirect in this feature has an i18n atm.